src/vector/_methods.py

```diff
@@ -3778,6 +3800,14 @@ def isclose(
             )
         return isclose.dispatch(rtol, atol, equal_nan, self, other)
 
+    def sum(self) -> VectorProtocol:
+        from vector._compute.lorentz import sum
+
+        return sum.dispatch(self)
+
+    def count_nonzero(self) -> ScalarCollection:
+        return self.lib.count_nonzero(self.tau2, axis=-1)
```

Comment:
This is a particular interpretation of `nonzero`. It would effectively count non-photons. Some insight into usage may be appreciated here!